api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -1100,6 +1100,18 @@ paths:
                 $ref: '#/components/schemas/V1QuestionsJsonList'
         '404':
           description: Not Found.
+  /signals/v1/private/sources/:
+    get:
+      description: Retrieve a list of signal sources.
+      responses:
+        '200':
+          description: List of signal sources
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/V1SourceList'
+        '404':
```

Comment:
Could also generate 401 and 403 if there are authentication / authorization problems.